pages/linux/genisoimage.md

```diff
@@ -0,0 +1,12 @@
+# genisoimage
+
+> Genisoimage is a pre-mastering program to generate ISO9660/Joliet/HFS hybrid filesystems.
+> More information: <https://manpages.debian.org/latest/genisoimage/genisoimage.1.en.html>.
+
+- Create an ISO image:
+
+`genisoimage -o {{myimage.iso}} {{path/to/directory}}`
+
+- Create an ISO image with files larger than 4GB:
+
+`genisoimage -o -allow-limited-size {{myimage.iso}} {{path/to/directory}}`
```

Comment:
Just out of curiosity, how does `-allow-limited-size` actually help here? The name of the option seems the opposite of what I'd expect (something like `-allow-unlimited-size`) :thinking:

Reply:
After thinking about it it's a bit confusing indeed
This option can be useful in situations where it is necessary to create an ISO image that contains more data than the maximum size specified by the standard ISO9660 (2 GB). 
If the option were named "-allow-unlimited-size", it would imply that there is no limit to the size of the ISO image that can be created, which is not true. In reality, this option only provides an alternative to work around the limit established by the ISO 9660 standard, but still within a general limit.

Reply:
So without the option, the limit is 2GB; with the option, what is the general limit?

Reply:
I realized that I put 4 in the description, I updated it to 2.

In general, the overall limit for the size of an ISO image is determined by the storage resources available on the device on which the image will be written and by the operating system and software used to read the image.

Reply:
Hmm. That's tricky to put into words (in a way that connects with the `-allow-limited-size` option name). I guess we have to just live with the awkwardness of the option name...

Reply:
Oh, wait, I think the manpage may help here:

> When processing files larger than 2GiB which cannot be easily represented in ISO9660, add them with a shrunk visible file size to ISO9660 and with the correct visible file size to the UDF system. [...]

So how about "Create an ISO image with files larger than 2GiB by reporting a smaller visible size"?

I'm not sure I understand the technical aspects of what is being added to the ISO9660 and UDF filesystems, to be honest, so the description I wrote may be misleading. Are both filesystems created when generating an ISO file?

Reply:
I believe that the description "Create an ISO image with files larger than 2GiB by reporting a smaller visible size in the ISO9660 file system and the correct actual size in the UDF file system." is a good way to explain what is happening behind the scenes when using this option.

Regarding your question, yes, both the ISO9660 file system and the UDF file system are created when an ISO image is generated with genisoimage. Genisoimage adds files larger than 2GB with a reduced visible size in the ISO9660 file system and with the correct actual size in the UDF file system. Therefore, the resulting ISO image will include both file systems.

Reply:
Hmm, ok, that's a bit clearer. But the description cannot be that long. I'm thinking perhaps "Create an ISO image with files larger than 2GiB by reporting a smaller apparent size for ISO9660", if that sounds correct to you. Otherwise, maybe we should just drop the intent to fully describe what's going on and just explain the final outcome and how to achieve it (i.e. keep the description as it currently is). Let me know what you think is more reasonable!

Reply:
Thank you for the suggestion! I have no objections to using 'Create an ISO image with files larger than 2GiB by reporting a smaller apparent size for ISO9660' as the command description. I will make the necessary changes in my PR to reflect this update. Thanks again for the help!

Reply:
No, thank you for the contribution and the patience with the review process :)